config/default.conf

```diff
@@ -30,7 +30,7 @@ url_repository = ""
 url_transformation = ""
 url_registration = ""
 
-redirect_to_website = "http://www.copyrighthub.org/"
+redirect_to_website = "https://openpermissions.org"
```

Comment:
And sort of related to https://github.com/openpermissions/onboarding-srv/pull/3#discussion-diff-62204431 ... what happens with the resolution service if we start having hubkeys with openpermissions as the resolver as well as copyrighthub?